src/segmentmytif/main.py

```diff
@@ -29,8 +32,15 @@ def read_input_and_labels_and_save_predictions(
     compute_mode: Literal["normal", "parallel", "safe"] = "normal",
     chunks: dict = None,
     chunk_overlap: int = DEFAULT_CHUNK_OVERLAP,
+    logger_root: logging.Logger = None,
     **extractor_kwargs,
 ) -> None:
+    # Hijack the global logger if a logger_root is provided
+    # This is designed to be used in QGIS environment
+    if logger_root is not None:
+        global logger
```

Comment:
you told me there were issues with collisions with other variables using this name, weren't you? Are they solved somehow now?